app/views/data_access/retrieval.html.erb

```diff
@@ -14,9 +14,9 @@
   <nav class="data-access-nav">
     <a href="<%=edsc_path "/data/configure?#{@retrieval.source}" %>"><i class="fa fa-arrow-circle-o-left"></i> Back to Data Access Options</a>
   </nav>
-  <header class="data-access-header">
-    <h1>Success!</h1>
-    <p>Your request has been processed.  See below for information on accessing your data.</p>
+  <header class="data-access-header" data-bind="visible: ui.projectList.collectionsToDownload().length == 0">
+    <h1>Request in Progress</h1>
+    <p>Your request is being processed.  Results will appear shortly.</p>
```

Comment:
@mreese84 Alright, have new changes in - let me know what you think.  I added a bit of padding so that the data-access isn't riding up on the back to collections links, updated the text to a more less-promising 'Please standby.', and also found other visibility cases to hide the header for when data-access-container has content.

I checked and it does now work for those ESI collections (I used the same collection you did), but I couldn't find one that'd fail.  Can you